array/models/actions/ActionGetContactByName.model.bxb

```diff
@@ -0,0 +1,11 @@
+action (ActionGetContactByName) {
+  description (fetch contact structure by contact name)
+  type (Fetch)
```

Comment:
Usually fetch actions are used on lazy source properties. I think simpler to just rename this to FindContact and use "Search" type

Reply:
Agree and done.